src/pages/DebuggerContent.tsx

```diff
@@ -156,7 +157,9 @@ const DebuggerContent = () => {
                         }
 
                         if (!error && program) {
-                          const maybeSpi = tryAsSpi(program.slice());
+                          const maybeSpiWithMetadata = tryAsSpi(program.slice(), true);
+                          const maybeSpiWithoutMetadata = tryAsSpi(program.slice(), false);
+                          const maybeSpi = maybeSpiWithMetadata || maybeSpiWithoutMetadata;
```

Comment:
```suggestion
                          const maybeSpi = maybeSpiWithoutMetadata || maybeSpiWithMetadata;
```

without metadata should take priority imho